src/App.css

```diff
@@ -17,6 +17,20 @@ body {
   position: relative;
 }
 
```

Comment:
| Zeplin's Design | @kifle23 Design |
| --- | --- |
| ![image](https://github.com/kifle23/Bookstore/assets/93039359/6930d6c0-7b58-4d86-ab8f-30435092d79a) | ![image](https://github.com/kifle23/Bookstore/assets/93039359/9f775fc3-23be-4735-b790-3cd8b8dfc2be) |

- [ ] It would be nice if you follow Zeplin's design and make your circular progress bar and other details equally spaced from the middle border line 👍 to make your project professional 😊